src/stacks-hierarchy/Root_TabNavigatorStack/Root_TabNavigatorStack.tsx

```diff
@@ -68,9 +68,18 @@ export const Root_TabNavigatorStack = ({navigation}: Props) => {
     fareContracts,
     serverNow,
   );
-  const {status: notificationStatus} = usePushNotifications();
+
+  const {register: registerForNotifications, status: notificationStatus} =
+    usePushNotifications();
```

Comment:
Should `usePushNotications` be a Context instead of a hook? Hooks doesn't have shared state between invocations, so every usage of it has its own status etc. By first glance, I am not exactly sure if `notificationStatus` can be anything other than `loading` in this file 🤔

Reply:
Maybe 🤔 This doesn't seem correct at least.